src/main/java/harvard/Ui.java

```diff
@@ -0,0 +1,131 @@
+package harvard;
+
+import harvard.exceptions.HarvardException;
+import harvard.tasks.Task;
+
+/**
+ * The Ui class is responsible for handling the user interface of the Harvard application.
+ * It provides methods to display messages and interact with the user.
+ */
+public class Ui {
+
+    /**
+     * Constructs a new Ui instance.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Displays a welcome message when the application starts.
+     */
+    public void showWelcome() {
+        String initial = "____________________________________________________________\n"
```

Comment:
Can consider using a string line break constant instead of doing "_______" which can help to make code cleaner